wolfcrypt/src/ecc.c

```diff
@@ -12805,6 +12835,9 @@ int wc_ecc_encrypt_ex(ecc_key* privKey, ecc_key* pubKey, const byte* msg,
             case ecAES_256_CTR:
             {
         #if !defined(NO_AES) && defined(WOLFSSL_AES_COUNTER)
+                byte ctr_iv[16];
```

Comment:
Is there an enum or macro we can use instead of hard coded 16?

Reply:
Fixed.